package.json

```diff
@@ -48,6 +48,7 @@
     "jsxgettext-recursive": "0.0.5",
     "load-grunt-tasks": "3.5.0",
     "mkdirp": "0.5.1",
+    "moment-timezone": "0.5.4",
```

Comment:
This looks to me like a hard dependency instead of a devDependency. Can you move this to `dependencies` and re-run shrinkwrap?